doc/effects.md

```diff
@@ -65,6 +65,24 @@ square.addEffect(RotateEffect(
   curve: Curves.easeInOut,
 ));
 ```
+
+## SequenceEffect
+
+This effect is a combination of other effects. You provide it with a list of your predefined effects, that are not added to a component.
```

Comment:
The ending on this phrase is a little confusing for, what do you mean by not added to a component?

Reply:
That they haven't called:
`myComponent.addEffect(effectThatIsPartOfSequence)`
Maybe I can clarify that.

Reply:
Maybe we can verify that on an assert as well? Not sure if is possible, just a thought

Reply:
Tried to clarify it, is it more clear now? Or should I simply remove that sentence?

Reply:
Added an assert for that too